db/migrate/20150701155145_add_last_url_to_users.rb

```diff
@@ -0,0 +1,7 @@
+class AddLastUrlToUsers < ActiveRecord::Migration
+  tag :predeploy
+  def change
+    add_column :users, :last_url, :string
```

Comment:
On Thu, Jul 02, 2015 at 08:56:49AM -0700, Brian Sadler wrote:
> I'm just going to assume that you looked into the object model and that users is the best object for last_url and last_url_title rather than just the first, easiest option.

Where else would you put it? The point of moving to the
database is so it is tied to a user instead of to a browser...
so the natural place of data tied to a user is, of course,
the user!